src/main/groovy/myFirstApp/MyAppRouter.groovy

```diff
@@ -0,0 +1,20 @@
+package myFirstApp
+import com.google.inject.Inject
+import org.restlet.Context
+import restling.restlet.RestlingRouter
+class MyAppRouter extends RestlingRouter{
+
+    @Inject
+    MyAppRouter(Context context) {
+        super(context)
+    }
+
+    @Override
+    void init() {
+        attach("/ping", Ping)
+        attach("/tasks/view",TaskController)
```

Comment:
usually REST urls should have nouns and not verbs.The verbs are the HTTP methods